src/query/ast/src/parser/token.rs

```diff
@@ -94,7 +96,7 @@ impl<'a> Iterator for Tokenizer<'a> {
 }
 
 #[allow(non_camel_case_types)]
-#[derive(Logos, Clone, Copy, Debug, PartialEq, Eq, Hash)]
+#[derive(Logos, EnumIter, Clone, Copy, Debug, PartialEq, Eq, Hash)]
```

Comment:
There is no need `EnumIter` here. We can iter enum by default:
https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=5d7b845c314a3ba541fcfec84f6778f9